live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/mapper/property/PropertiesMapperTest.java

```diff
@@ -0,0 +1,232 @@
+package org.alfresco.hxi_connector.live_ingester.messaging.in.mapper.property;
```

Comment:
Let's add the same copyright headers to test files too. If the build isn't failing then let's update the plugin to scan all code files.

Reply:
meh, I am using two different workspaces and I've set copyright up just on one of them :/ build is failing so is ok from this side

Reply:
fixed